demo/src/lib/gtagEvents.js

```diff
@@ -0,0 +1,132 @@
+import {v4} from 'uuid';
+import {event} from './gtag.js';
+import {getItemParameters, getItemsArrayFromCart, computePriceOfItemsInCart} from '../utils.js';
+
+/**
+ * Sends a select_item event to Google Analytics.
+ * View the [Ecommerce Guide](https://developers.google.com/analytics/devguides/collection/app-web/ecommerce)
+ * to review the event and its accepted parameters.
+ * @param {string} itemId
+ */
+export function sendSelectItemEvent(itemId) {
+  event('select_item', {
+    items: [{
+      ...getItemParameters(itemId),
+    }],
+  });
+}
```

Comment:
nit: This pattern is repeated multiple times. Maybe DRY it up by creating a module-level helper function?